### DISCUSSION TITLE
[Question] Location of deb installer translation

### DISCUSSION THREAD

**Discussion by ovari** [evilsocket/opensnitch#788] [General]
# opensnitch_1.4.0.rc2-1_amd64.deb

From the deb installer, where do we add the Hungarian translation for the information below
>**GNU/Linux firewall application**
OpenSnitch is a GNU/Linux firewall application. Whenever a program makes a connection, it'll prompt the user to allow or deny it.
The user can decide if block the outgoing connection based on properties of the connection: by port, by uid, by dst ip, by program or a combination of them.
These rules can last forever, until the app restart or just one time.
The GUI allows the user to view live outgoing connections, as well as search by process, user, host or port. 

which is from the image below
![opensnitch_1.4.0.rc2-1_amd64.deb](https://user-images.githubusercontent.com/17465872/119224213-c3127580-bb40-11eb-8fa2-3fd1d6d19b99.png)

# python3-opensnitch-ui_1.4.0.rc2-1_all.deb

From the deb installer, where do we add the Hungarian translation for the information below
>**opensnitch application firewall GUI**
opensnitch-ui is a GUI for opensnitch written in Python. It allows the user to view live outgoing connections, as well as search for details of the intercepted connections.
The user can decide if block outgoing connections based on properties of the connection: by port, by uid, by dst ip, by program or a combination of them.
These rules can last forever, until restart the daemon or just one time. 

which is from the image below
![python3-opensnitch-ui_1.4.0.rc2-1_all.deb](https://user-images.githubusercontent.com/17465872/119224543-37014d80-bb42-11eb-8adb-0be364dc5e48.png)

Thank you

**Comment by gustavo-iniguez-goya** [comment]
I think that the translation of that text is done in debian/control file, with fields like Description-hu, Description-es, etc.

I'll test it and I'll let you know if it works.

**Comment by ovari** [comment]
Have you been able to test?

It would be nice to have the Hungarian translations available for the 1.4.0 release.

Thank you

![kép](https://user-images.githubusercontent.com/17465872/127106537-db1440b2-bd77-4a56-8fcb-674b59ea88e9.png)

**Comment by gustavo-iniguez-goya** [comment]
no @ovari , not yet. Actually I completely forgot about this, thank you for reminding me it.

**Comment by ovari** [comment]
@gustavo-iniguez-goya are you able to provide an update on this issue? Thank you

**Comment by gustavo-iniguez-goya** [comment]
I haven't taken a look at this. It's low priority right now.
